### ISSUE TITLE
[New Language Support]: French

### ISSUE THREAD

**Issue by Drizztfire** [PRProd/HA-Firemote#370]
### Language Code

fr

### Translation Chart

```text
01. App Launch Button => Bouton de lancement d'application
02. App Launcher Relative Scale => Échelle relative du lanceur d'applications
03. Apple Computers => Ordinateurs Apple
04. Apple Music => Apple Music
05. Apple Photos => Apple Photos
06. Associated => Associé
07. bottom => bas
08. Compatibility Mode => Mode de compatibilité
09. Computers => Ordinateurs
10. Custom Remote Skin => Apparence personnalisée de la télécommande
11. Default for => Par défaut pour
12. Device Family => Famille d'appareils
13. Device Model => Modèle d'appareil
14. Device Name Text Color => Couleur du texte du nom de l'appareil
15. DPad Style => Style du DPad
16. Entity => Entité
17. Function: Captions => Fonction : Sous-titres
18. Function: Channel + => Fonction : Chaîne +
19. Function: Channel - => Fonction : Chaîne -
20. Function: Control Center => Fonction : Centre de contrôle
21. Function: Find My Remote => Fonction : Trouver ma télécommande
22. Function: Mute => Fonction : Muet
23. Function: Next => Fonction : Suivant
24. Function: Pairing => Fonction : Appairage
25. Function: Previous => Fonction : Précédent
26. Function: Reboot => Fonction : Redémarrer
27. Function: Search => Fonction : Recherche
28. Function: Settings => Fonction : Paramètres
29. Function: Skip Backward => Fonction : Retour rapide
30. Function: Skip Forward => Fonction : Avance rapide
31. Function: Switch Apps => Fonction : Changer d'application
32. Function: System Update => Fonction : Mise à jour du système
33. Function: Volume + => Fonction : Volume +
34. Function: Volume - => Fonction : Volume -
35. hidden => caché
36. Hide frames around button groups => Masquer les cadres autour des groupes de boutons
37. Music => Musique
38. Name Position => Position du nom
39. optional => optionnel
40. Photos => Photos
41. Remote Style => Style de la télécommande
42. Scale => Échelle
43. Settings => Paramètres
44. Search => Recherche
45. top => haut
46. Visible Device Name => Nom de l'appareil visible
```

**Comment by PRProd** [comment]
Greetings @Drizztfire!

Thank you for this request!  Before I publish this translation into a beta release, is there anything you can see from this image that is not quite right?  You are the expert here, so I will make any changes that you suggest:

![image](https://github.com/PRProd/HA-Firemote/assets/25486599/af55c23c-f7ef-415f-815f-5cb027646ccc)

(Please note: this image does not cover every translation that you provided - other Firemote option menus use different parts)

**Comment by Drizztfire** [comment]
It seems good !

The only one that bugs me out is this one : 
![image](https://github.com/PRProd/HA-Firemote/assets/7252652/c60dd323-cb17-4512-8764-b00d6354281b)
If it's under your control it should probably be : 
Entité Android TV Remote Associé
And also the one over it :
Entité Android Debug Bridge

Everything else seems ok

Thanks

**Comment by PRProd** [comment]
@Drizztfire ~

I have published Firemote Beta Version [v3.3.3b1](https://github.com/PRProd/HA-Firemote/releases/tag/v3.3.3b1) which contains the requested French translations.  Please download this beta, and go through the Firemote card menus for yourself to see if there is anything else that should be adjusted.  Thanks!

---

After updating to the newest version, if you are unable to see the new updates on your Firemote card, please visit [this link](https://github.com/PRProd/HA-Firemote/wiki/Force-a-Refresh) for help.
If you are experiencing issues with seeing changes related to a language translation, please visit [this link](https://github.com/PRProd/HA-Firemote/wiki/Changing-Your-Home-Assistant-Language) for help.

**Comment by PRProd** [comment]
@Drizztfire ~

Did you have a chance to look at the beta version?  I typically wait to publish language translations to the official Firemote version until I receive word from the translator (you) that they look OK.  Changes can still be made after it is published too, so this isn't a required step.
